packages/flutter_tools/lib/src/commands/ios.dart

```diff
@@ -122,21 +163,31 @@ class IOSCommand extends FlutterCommand {
     // Step 2: Inflate the archive into the user project directory
     bool result = await _inflateXcodeArchive(xcodeprojPath, archiveBytes);
     if (!result) {
-      printError("Error: Could not init the Xcode project: the 'ios' directory already exists.");
-      printError("To proceed, remove the 'ios' directory and try again.");
-      printError("Warning: You may have made manual changes to files in the 'ios' directory.");
+      printError("Could not inflate the Xcode project archive.");
       return 1;
     }
 
-    // Step 3: Populate the Local.xcconfig with project specific paths
+    // Step 3: The generated project should NOT be checked into the users
+    //         version control system. Be nice and write a gitignore for them if
+    //         one does not exist.
+    File generatedGitignore = new File(path.join(iosFilesPath, ".gitignore"));
```

Comment:
We might instead ignore `.gitignore/Generated/` in the top level git ignore file created in the `create` command. There's also a path there and in the `.analysis_options` file that should be updated w/ the new ios paths.